primitives/src/vid/advz/precomputable.rs

```diff
@@ -141,6 +144,26 @@ where
             commit,
         })
     }
+
+    fn is_consistent_precompute(
+        commit: &Self::Commit,
+        precompute_data: &Self::PrecomputeData,
+        payload_byte_len: u32,
+        num_storage_nodes: u32,
```

Comment:
Presumably both committer and disperser have initialized the VID scheme with the same `payload_byte_len` and `num_storage_nodes` (otherwise, the pre-computable trait would make little sense). If this is not the case then this new verification check will also reveal that, no?